### ISSUE TITLE
About train losses and evalution parameters setting

### ISSUE THREAD

**Issue by MrCrims** [FoundationVision/LlamaGen#56]
Hello, I'm confused about my traing on GPT-XL image size 384×384. After 300 epoches, the traning loss is around 6.9, and FID is just 6.8. Could you share your detailed training settings? 
Besides, when I try to use c2i_XL_384.pt to reproduce your results, my best results is 2.73 different with 2.62 in the paper. Could you share your detailed evalution settings including cfg-scale ,etc ?
I will be grateful if you could help me.

**Comment by Alpha-X-95** [comment]
Would you like to show us the training loss for c2i and t2i.
